Shared/Realm.Sync.Shared/Exceptions/HttpException.cs

```diff
@@ -29,16 +29,19 @@ public class HttpException : Exception
         /// <summary>
         /// Gets the <see cref="HttpStatusCode"/> of the response.
         /// </summary>
+        /// <value>A well known <see cref="HttpStatusCode"/>.</value>
         public HttpStatusCode StatusCode { get; }
 
         /// <summary>
         /// Gets the ReasonPhrase of the response.
         /// </summary>
+        /// <value>The ReasonPhrase of the response.</value>
```

Comment:
This, as well as the summary, provides very little information about what this actually is. Maybe it could say "HTTP ReasonPhrase" or something?